runtime/runtime.go

```diff
@@ -27,7 +27,10 @@ var (
 
 	failpoints   map[string]*Failpoint
 	failpointsMu sync.RWMutex
```

Comment:
With 2 mutexes, can you comment what each of them does. My understanding is:
* `failpointsMu` - prevents concurrent accesses to failpoints variable
* `panicMu` - prevents executing executing panic failpoint while serving a http request by gofail